src/main/controllers/service/ChooseServiceAreaController.ts

```diff
@@ -77,7 +77,7 @@ export class ChooseServiceAreaController {
           res.redirect('/services/unknown-service');
         }
       } else {
-        res.redirect('/services/unknown-service');
+        res.redirect('/services/' + req.params.service + '/' + req.body.serviceArea + '/search-by-postcode');
```

Comment:
if you want aol you can attach ?aol= on that endpoint. You can get aol from `serviceArea`